app/testsuite/network_config.go

```diff
@@ -48,7 +48,7 @@ func DefaultConfig() network.Config {
 		BondDenom:         sdk.DefaultBondDenom,
 		MinGasPrices:      fmt.Sprintf("0.000006%s", sdk.DefaultBondDenom),
 		AccountTokens:     sdk.TokensFromConsensusPower(1000, sdk.DefaultPowerReduction),
-		StakingTokens:     sdk.TokensFromConsensusPower(1000, sdk.DefaultPowerReduction),
+		StakingTokens:     sdk.TokensFromConsensusPower(1e7, sdk.DefaultPowerReduction),
```

Comment:
What is this change for?

Reply:
Making sure accounts have enough tokens to create classes in integration tests. Should we top up the accounts some other way instead?